BossMod/ActionQueue/ActionDefinition.cs

```diff
@@ -86,7 +100,8 @@ public float ChargeCapIn(ReadOnlySpan<Cooldown> cooldowns, int level)
         if (MainCooldownGroup < 0)
             return 0;
         var cdg = cooldowns[MainCooldownGroup];
-        return cdg.Total > 0 ? (MaxChargesAtLevel(level) * Cooldown - cdg.Elapsed) : 0;
+        // TODO MaxChargesAtLevel is zero for some actions with charges (like Air Anchor), but cdg.Total can only be nonzero if the action is on cooldown, i.e. if it was used, i.e. it has more than 0 charges
+        return cdg.Total > 0 ? (Math.Max(1, MaxChargesAtLevel(level)) * Cooldown - cdg.Elapsed) : 0;
```

Comment:
I think MaxChargesAtLevel shouldn't ever return 0... Is that because sheets have 0? Probably better to clamp it to 1 there.

Reply:
it's 0 for probably 99% of actions, like GCDs that don't have a charge system, but it's *also* zero for single-charge GCDs that also have their own separate cooldown. one example is machinist Air Anchor. does it make sense for regular GCDs to default to having 1 max charge?

Reply:
Yeah i think so, normally action has one charge.